src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -3849,4 +3854,62 @@ public Response getExternalToolDVParams(@Context ContainerRequestContext crc,
             return wr.getResponse();
         }
     }
+
+    @GET
+    @Path("summaryFieldNames")
```

Comment:
I started a thread in Slack about being consistent in how we expose database settings: https://iqss.slack.com/archives/C010LA04BCG/p1686603802872139

This might be fine but we should make an active decision because we'll be doing this a lot for the SPA. It needs to know how the backend is configured.